package chapter10_2;

public class ExceptionExample02 {
	public static void main(String[] args) {
		int[] scores = new int[3];
		scores[0] = 70;
		scores[1] = 80;
		scores[2] = 95;
		scores[3] = 50;
//		要素数３の配列に４つ目の値を入れようとすると例外が投げられる。
	}

}
